drive/types.bal

```diff
@@ -56,41 +56,41 @@ public type About record {
     boolean canCreateDrives?;
 };
 
-# File Record Type
+# Represents File.
 @display {label: "File"}
 public type File record {
     *FileMetadata;
     *GeneratedFileMetadata;
 };
 
-# File Metadata Record Type
+# Represents file metadata.
```

Comment:
Where is this description used?

Reply:
It's internally used. Not displayed on Choreo